src/client/App.jsx

```diff
@@ -8,12 +8,17 @@ import SignInPage from "./Pages/SignInPage";
 import FriendsPage from "./pages/FriendsPage";
 
 export default function App() {
+  const handleSubmit = e => {
+    e.preventDefault();
+    console.log("clicked");
+  };
+
   return (
     <Router>
       <HomePage path="/" />
       <TestPage path="/test" />
       <SignUpPage path="/sign-up" />
-      <Feed path="/feed" />
+      <Feed path="/feed" onSubmit={handleSubmit} />
```

Comment:
Rename this to something else, because each component in React can have an onSubmit prop and this may cause unwanted errors. The value will still be handleSubmit though.

Reply:
done